api/test/authentication/auth.spec.ts

```diff
@@ -32,26 +32,24 @@ describe('AppController (e2e)', () => {
   });
 
   afterAll(async () => {
-    await Promise.all([app.close()]);
+    await app.close();
   });
 
   describe('Authentication', () => {
-    beforeAll(
-      async (): Promise<User> => {
-        const user: User = new User();
-        user.email = E2E_CONFIG.users.basic.aa.username;
-        user.fname = 'a';
-        user.lname = 'a';
-        user.displayName = 'User A A';
-        user.salt = await bcrypt.genSalt();
-        user.password = await bcrypt.hash(
-          E2E_CONFIG.users.basic.aa.password,
-          user.salt,
-        );
-        user.isActive = true;
-        return user.save();
-      },
-    );
+    beforeAll(async (): Promise<User> => {
+      const user: User = new User();
```

Comment:
I think creating this mock user should not be placed in the beforeAll as it is not required for all the auth related use cases

Reply:
IIRC this is out of scope of this PR, it's just a code style change